code_search/kubeflow/components/t2t-job.libsonnet

```diff
@@ -84,7 +84,7 @@ local baseParams = std.extVar("__ksonnet/params").components["t2t-job"];
     local workerEnv = [
       {
         name: "GOOGLE_APPLICATION_CREDENTIALS",
-        value: "/secret/gcp-credentials/user-gcp-sa.json"
+        value: "/secret/gcp-credentials/user-gcp-sa.json",
```

Comment:
can you add another env variable where you point to log dir? Then we could refer to it in code and allow everyone to use tensorboard with this example

Reply:
Can you explain what you are asking? Everyone can use tensorboard with this example. They just configure their TB job to point at the same directory that their T2T job writes the model.

Reply:
I was thinking of allowsing something like `tf.summary.FileWriter(os.getenv("KUBEFLOW_LOGDIR"), sess.graph))`

Reply:
Why do we need that? The trainer code takes command line argument modelDir which controls where the model and event files are written. So if the user wants to change the location of the event files. They can just change that.

Reply:
ahh that makses sense. I've missed that.